backend/attendance-service/src/main/java/org/egov/config/AttendanceServiceConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package org.egov.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.egov.tracer.config.TracerConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Import;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
+@Import({TracerConfiguration.class})
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class AttendanceServiceConfiguration {
+
+    // kafka topics
+    @Value("${att.svr.log.kafka.create.topic}")
```

Comment:
Follow a common naming convention. In the other 2 PRs "attendance.service" is used. Use the same here.

Reply:
Updated as per the common naming convention.